node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/v2/responses/node4-after-changes.json

```diff
@@ -39,6 +39,7 @@
   "failCount": 0,
   "hardwareFailure": true,
   "hardwareFailureType": "memory_mcelog",
+  "hardwareFailureDescription": "memory_mcelog",
```

Comment:
So it might not be feasible to fill node repo with nodes that haven't been patched.

In any case, the field key doesn't change, which makes the change fine - no test needs to be added.